cpp/Contact.h

```diff
@@ -338,12 +308,16 @@ class Contact
     /// be padded to 3D, (shape (num_nodes, 3)).
     kernel_fn<PetscScalar> unbiased_rhs
         = [num_coordinate_dofs, gdim, tdim, ref_jacobians, dphi_c,
-           facet_normals, phi, dphi, qweights, ndofs_cell, offsets, affine,
-           update_jacobian, update_normal, bs, num_q_points](
+           facet_normals, phi, dphi, q_weights, ndofs_cell, offsets, affine,
+           update_jacobian, update_normal, bs, qp_offsets](
               std::vector<std::vector<PetscScalar>>& b, const PetscScalar* c,
               const PetscScalar* w, const double* coordinate_dofs,
               const int facet_index, const std::size_t num_links)
     {
+      // assumption that the vector function space has block size tdim
```

Comment:
Should this comment be here? We should assert this when generating the kernel, i.e.
```c++
if (element_bs != tdim)
    throw std::invalid_argument("Function space not vector space of dimension gdim");
```